server/services/store/sqlstore/category.go

```diff
@@ -172,6 +172,8 @@ func (s *SQLStore) categoriesFromRows(rows *sql.Rows) ([]model.Category, error)
 
 	for rows.Next() {
 		category := model.Category{}
+		var nullableSortOrder *int
```

Comment:
Although the approach works fine, I'd suggest to change it for consistency and do the same thing we've done in other tables, which is to create a `categoryFields` method to return a `[]string` with all the fields and use it in all methods that later call `categoriesFromRows`

In those fields you can use `COALESCE` to set the default value, so the scan code can reliably expect a value set for the nullable column

Reply:
I agree - this would be consistent with the other store methods.

Reply:
Thanks for pointing this out. Done.